docs/interfaces/lite/getting_started.md

```diff
@@ -1,20 +0,0 @@
-# Getting Started
```

Comment:
I removed all docs from this folder. They mentioned LCD, gaia-lite, which I believe is all removed.

Reply:
cc @marbar3778 @alexanderbez : Are we okay to remove all docs from this folder?